eng/pipelines/smoke-tests.yml

```diff
@@ -39,7 +41,39 @@ jobs:
           packageType: sdk
           version: "$(DotNetCoreSDKVersion)"
 
-      - script: dotnet run -p ./common/SmokeTests/SmokeTest/SmokeTest.csproj --framework $(TestTargetFramework)
+      - pwsh: Register-PackageSource -Name NightlyFeed -Location '$(BlobFeedUrl)' -ProviderName Nuget
```

Comment:
Might be worth just doing this inside the script so we don't have to have these multiple steps if we try to run this locally.